lib/API/RelationService.php

```diff
@@ -61,4 +61,32 @@ public function loadFieldRelationLocations(
         array $contentTypeIdentifiers = [],
         ?int $limit = null,
     ): array;
+
+    /**
+     * Load all reverse related Content from $fieldDefinitionIdentifier field to the given
+     * $content, optionally limited by a list of $contentTypeIdentifiers and $limit.
+     *
+     * @return \Netgen\IbexaSiteApi\API\Values\Content[]
+     */
+    public function loadReverseRelations(
```

Comment:
@emodric Please rename to `loadReverseFieldRelations`, here and similarly on other places as well.

Reply:
Done in 1fa482ebfe4e8300e41cc8b9ab07b57105a30e08